Infrastructure/Services/AuthService.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System.Globalization;
```

Comment:
Overall, the code is concise and well-organized. The dependencies are injected through the constructor, and the methods have clear and straightforward responsibilities. One suggestion would be to add error handling to the public methods to provide better feedback to the calling code.

Reply:
Yeah, good point need to create separate task with Error Handling approach, where we should agreed on next topics
1. Throw exception vs Functionality way(`Language.Ext`) 
2. Global and local errors scope
3. Error responses format